src/desktop/apps/artwork/components/commercial/templates/price.jade

```diff
@@ -1,6 +1,6 @@
 - var isPermanentCollection = artwork.availability === 'permanent collection'
 - var isOnLoan = artwork.availability === 'on loan'
-- var showPriceLabel = artwork.price && artwork.availability !== 'sold' && !isOnLoan && !isPermanentCollection
+- var showPriceLabel = (artwork.is_acquireable || artwork.is_offerable) && artwork.availability !== 'sold' && !isOnLoan && !isPermanentCollection
```

Comment:
I'm not sure this is right...there's other reasons to show a price (such as when it's set as a range), in which case it won't be acquireable or offerable (I think).

Why not just add this guard near/as a specific shipping information guard?

Reply:
Good point! I'll move the guard further down. Still getting familiar with all of the different artwork states.